rmf_traffic_ros2/include/rmf_traffic_ros2/StandardNames.hpp

```diff
@@ -53,6 +53,8 @@ const std::string NegotiationForfeitTopicName = Prefix +
   "negotiation_forfeit";
 const std::string NegotiationConclusionTopicName = Prefix +
   "negotiation_conclusion";
+const std::string NegotiationStatusTopicName =  Prefix +
```

Comment:
Is this prefix added to all topics published by this node? If so...

I can see this can't be changed now because all the lines above to the same thing, but putting a prefix on every topic is not a good idea. If some have the prefix and some don't, that's fine, but that doesn't appear to be the case here. It is better to let the launch system add a namespace to all the topics. Hard-coding the prefix circumvents the launch system, reducing flexibility.

Reply:
The prefixing was my idea, about a year ago. The motivation was to make it explicit that these topics are internal to the traffic management/negotiation system and probably shouldn't be touched by anything outside of that.

If we wanted to further namespace these topics (e.g. to have two independent traffic management/negotiation systems on a single network) would that be something that can still be done with the launch system, or does explicitly having this namespace somehow prohibit that?

Reply:
> make it explicit that these topics are internal to the traffic management/negotiation system

That's a sensible use of a prefix. I don't think the current name makes that obvious, though.

> If we wanted to further namespace these topics (e.g. to have two independent traffic management/negotiation systems on a single network) would that be something that can still be done with the launch system, or does explicitly having this namespace somehow prohibit that?

You can still add namespaces using the launch system. The topic names, even with the prefix, are still relative. Starting hard-coded topic names with a slash is the biggest no-no because that completely blocks remapping. Having a prefix just restricts it, for better or worse depending on the use case.